src/app/pages/MediaArticlePage/MediaArticlePage.styles.ts

```diff
@@ -86,4 +86,14 @@ export default {
         padding: `${spacings.DOUBLE}rem`,
       },
     }),
+
+  responsiveComponentWrapper: ({ spacings, mq }: Theme) =>
+    css({
+      marginBottom: `${spacings.TRIPLE}rem`,
+
+      [mq.GROUP_4_MIN_WIDTH]: {
+        marginBottom: `${spacings.FULL}rem`,
+        padding: `0 ${spacings.DOUBLE}rem ${spacings.DOUBLE}rem ${spacings.DOUBLE}rem`,
```

Comment:
Just checked with @emilymynall, this padding can be removed. The promos should take up the entire width available of the SecondaryColumn column.

Reply:
👍